Devscord.DiscordFramework/ReactionInformation.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Discord;
+using Discord.WebSocket;
+
+namespace Devscord.DiscordFramework
+{
+    public class ReactionInformation
+    {
```

Comment:
zrobiłbym to jako immutable

(spójrz jak wygląda to przy contextach, np UserContext)

Reply:
Czy ja wiem. Chyba lepiej jest tak jak jest teraz. W obecnym formacie ładnie wygląda wypełnianie obiektu tej klasy. Poza tym to w sumie struktura, ona nie musi niczego ukrywać przed światem. Jak jakaś metoda, która otrzyma taki obiekt, to może po prostu robić z tym co chce.

Reply:
moim zdaniem również lepiej dać jako immutable

Reply:
czyli musi być koniecznie jako immutable?

Reply:
Problem pojawia sie jak zapomnisz o ustawieniu którejś wartości
Albo jak ktoś pomyśli że można coś zmienić, bo przecież jest taka możliwość

Reply:
`Jeśli metoda otrzyma taki obiekt, to może po prostu robić z tym co chce`

No właśnie nie powinna móc robić wszystkiego, powinna móc tylko odczytywać
Może być sytuacja gdzie ten obiekt przekazujesz przez wiele miejsc

I w końcowym etapie chciałbyś mieć pewność, że wszystkie dane są prawidłowe - takie jakie były na początku, bez konieczności sprawdzania każdego wcześniejszego etapu 

Jak robisz coś sam to wiesz co autor miał na myśli... nie zawsze bo po dłuższym czasie sie takie rzeczy zapomina, ale mniej więcej kojarzysz o co chodziło 
Ale jak pracujesz w grupie, to dobrze byłoby informować innych jak coś powinno być używane 
Można zrobić to komentarzami, ale komentarze często zaśmiecają kod + jest szansa że ktoś ich nie zauważy 
Dlatego dużo lepszą opcją jest pisanie takiego kodu, żeby sam mówił do czego on służy

Reply:
`poza tym to w sumie struktura, ona nie musi niczego ukrywać przed światem`
Nie ma ukrywać, po prostu ma nie być edytowalna już po stworzeniu